etc/picongpu/juwels-jsc/gpus_picongpu.profile.example

```diff
@@ -11,29 +11,38 @@ export MY_NAME="$(whoami) <$MY_MAIL>"
 
 # Project Information ######################################## (edit this line)
 #   - project and account for allocation
-export proj=$(groups | awk '{print $4}')
-export account=$(jutil user projects -n | awk '{print $NF}')
+#   jutil user projects will return a table of project associations.
+#   Each row contains: project,unixgroup,PI-uid,project-type,budget-accounts
+#   We need the first and last entry.
+#   Here: select the last available project.
+export proj=$( jutil user projects --noheader | awk '{print $1}' | tail -n 1 )
+export account=$(jutil user projects -n | awk '{print $NF}' | tail -n 1)
 
 # Text Editor for Tools ###################################### (edit this line)
 #   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
 #export EDITOR="nano"
 
 # Set up environment, including $SCRATCH and $PROJECT
-jutil env activate -p $proj -A $account
+# Handle a case where the budgeting account is not set.
+if [ "$account" = "-" ]; then 
+    jutil env activate --project $proj;
+else
+    jutil env activate --project $proj --budget $account
+fi
 
 # General modules #############################################################
 #
 module purge
-module load GCC/8.3.0
-module load CUDA/10.1.105
-module load CMake/3.14.0
-module load MVAPICH2/2.3.3-GDR
-module load Python/3.6.8
-
-module load Boost/1.69.0-Python-3.6.8
-module load HDF5/1.10.5
+module load GCC/9.3.0
+module load CUDA/11.0
+module load CMake/3.18.0
+module load ParaStationMPI/5.4.7-1
```

Comment:
Is this MPI GPU aware?

Reply:
Yes, but by default the `mpi-settings/default` module will be loaded, which works fine but could be replaced by `mpi-settings/CUDA`.
I refrained from adding it, since I am not sure whether this module works OK on the _cluster_ (it does on the booster, though).

Reply:
Please add this option. Working with a non cuda aware MPI together with GPUs requires additional MPI flags to avoid the MPI is caching freed pointers.

Reply:
Alright. I added it as default.

Reply:
`mpi-settings/CUDA` should also be taken on the Cluster. It sets environment variables to tune PSMPI and especially UCX for GPU-awareness, for example `UCX_TLS=[…,]cuda[,…]`. The environment variables might not be identical on Cluster and Booster; but that's exactly the reason why the module exist – so we can care about this and not the user.